core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java

```diff
@@ -149,13 +149,14 @@ protected void encodeInput(final FacesContext context, final InputPhone inputPho
         final ResponseWriter writer = context.getResponseWriter();
         final String inputId = clientId + "_input";
         final String inputStyle = inputPhone.getInputStyle();
+        final String styleClass = createStyleClass(inputPhone, "inputStyleClass", InputText.STYLE_CLASS);
 
         writer.startElement("input", null);
         writer.writeAttribute("id", inputId, null);
         writer.writeAttribute("name", inputId, null);
         writer.writeAttribute("type", inputPhone.getType(), null);
         writer.writeAttribute("value", valueToRender, null);
-        writer.writeAttribute(Attrs.CLASS, createStyleClass(inputPhone, InputText.STYLE_CLASS), "styleClass");
+        writer.writeAttribute(Attrs.CLASS, styleClass, "styleClass");
```

Comment:
Shouldn't the third param be `"inputStyleClass"` (like line 152)?

Reply:
Yeah I guess it should be!

Reply:
Fixed!